README.md

```diff
@@ -49,3 +51,15 @@ NOTICE:  version "X.X.X" of extension "postgis_tiger_geocoder" is already instal
 ALTER EXTENSION
 ```
 
+### Adding custom locale
+
+If you need to use another locale, just generate a new Dockerfile with this content (this example is for italian locale):
+
+```
+FROM mdillon/postgis:11
```

Comment:
This should refeference postgis/postgis:latest image instead of mdillon/postgis:11